docs/concepts/schedules.md

```diff
@@ -22,28 +22,13 @@ Prefect Cloud can also schedule flow runs through event-driven [automations](/co
 
 Schedules tell the Prefect API how to create new flow runs for you automatically on a specified cadence.
 
-You can add schedules to any flow [deployment](/concepts/deployments/).
+You can add a schedule to any flow [deployment](/concepts/deployments/).
 The Prefect `Scheduler` service periodically reviews every deployment and creates new flow runs according to the schedule configured for the deployment.
 
-There are several recommended ways to create a schedule for a deployment:
+!!! tip "Support for multiple schedules"
+    We are currently rolling out support for multiple schedules per deployment. You can now assign multiple schedules to deployments in the Prefect UI, the CLI via `prefect deployment schedule` commands, the `Deployment` class, and in [block-based deployment](/concepts/deployments/#block-based-deployments) YAML files.
 
-- Through the Prefect UI
-- Via a the `cron`, `interval`, or `rrule` parameters if building your deployment via the [`serve` method](/concepts/flows/#serving-a-flow) of the `Flow` object or [the `serve` utility](/concepts/flows/#serving-multiple-flows-at-once) for managing multiple flows simultaneously
-- If using [worker-based deployments](/concepts/work-pools/)
-  - Through the interactive `prefect deploy` command
-  - With the `deployments` -> `schedules` section of the `prefect.yaml` file )
-
-## Creating schedules through the UI
-
-You can add a schedule on the **Deployments** tab of the UI.
-
-Under **Schedules** click the **+** button.
-Then select **Interval** or **Cron** to create a schedule.
-
-![Prefect UI with Interval button selected](/img/ui/interval-schedule.png)
-
-Once a schedule has been created, a number of scheduled flow runs will be visible in the UI.
-The schedule is viewable in human-friendly text on the **Deployments** page.
+    Support for multiple schedules in `flow.serve`, `flow.deploy`, `serve`, and [worker-based deployments](/concepts/work-pools/) with `prefect deploy` will arrive soon.
```

Comment:
Serve support PR has at least one approval, so likely to be in this release, too. #12107

Reply:
Oh, good point!

Reply:
I'm going to wait and we can update that if we do indeed ship `flow.serve` support, which is very likely given the PR... but not merged yet.